eternalcore-core/src/main/java/com/eternalcode/core/updater/UpdaterController.java

```diff
@@ -0,0 +1,57 @@
+package com.eternalcode.core.updater;
+
+import com.eternalcode.core.configuration.implementation.PluginConfiguration;
+import net.kyori.adventure.audience.Audience;
+import net.kyori.adventure.platform.AudienceProvider;
+import net.kyori.adventure.text.minimessage.MiniMessage;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+
+import java.util.concurrent.CompletableFuture;
+
+public class UpdaterController implements Listener {
```

Comment:
Please Add `@FeatureDocs` annotation, for more explain: 
![image](https://github.com/EternalCodeTeam/EternalCore/assets/65517973/acc7cbae-7659-484a-99c3-f27ac1b6ba1c)

Or just join voice channel to me <3

Reply:
https://docs.eternalcode.pl/eternalcore/features.html